### ISSUE TITLE
ObservedResults sortDescriptor isn't a sequence

### ISSUE THREAD

**Issue by PhilippeWeidmann** [realm/realm-swift#7709]
### How frequently does the bug occur?

All the time

### Description

`@ObservedResults` initializer documentation states that the sortDescriptor argument is a sequence. 
However it only takes one `SortDecriptor` object as an argument. 

I don't know if the documentation is wrong or if the feature is missing.
Either case it would be great if it could take an array of sortDescriptor to do a `sorted(by: [])` with multiple descriptors.

### Can you reproduce the bug?

Yes, always

### Version

10.24.1

### What SDK flavour are you using?

Local Database only

### Are you using encryption?

No, not using encryption

### Platform OS and version(s)

iOS 15.4

### Build environment

Xcode version: 13.3
Dependency manager and version: SPM

**Comment by pavel-ship-it** [comment]
Hi @PhilippeWeidmann 
Thank you for finding. This is a missing part of API

**Comment by tangowhisky-dev** [comment]
Any update on this. I'm using latest RealmSwift 20.0.2 and still facing the same issue when trying to pass multiple sortdescriptors to ObservedResults.

**Comment by aehlke** [comment]
@tangowhisky-dev realm is dead

**Comment by ryangittings** [comment]
It’d be great to have multiple sort in the ObservedResults.

**Comment by aehlke** [comment]
@ryangittings Realm will have no more improvements sadly

**Comment by ryangittings** [comment]
Such a shame. I can see some recent commits though?

**Comment by aehlke** [comment]
@ryangittings one last mongodb employee still takes care of OS update compatibility on a volunteer basis. there is zero feature development or other maintenance and no community effort. you should probably switch to SQLiteData https://github.com/pointfreeco/sqlite-data

**Comment by ryangittings** [comment]
Such a shame! Could Realm be forked at all?

**Comment by aehlke** [comment]
> Such a shame! Could Realm be forked at all?

Yes but there's no community momentum, you will be on your own. It's too large and complex.
